client/next.js

```diff
@@ -10,13 +10,17 @@ const {
   __NEXT_DATA__: { component, props, ids, err }
 } = window
 
-const Component = evalScript(component).default
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
We need to wait app loads some webpack stuff. This won't make any performance issues either.
If we need to support older IE version, we may have to do more.
Do we?